src/main/java/org/reaktivity/reaktor/internal/agent/ElektronAgent.java

```diff
@@ -221,6 +226,51 @@ public ElektronAgent(
         this.groupId = initial;
     }
 
+    private static class ResolverRef implements RouteManager
+    {
+        final ThreadLocal<Resolver> resolver;
+
+        ResolverRef(Supplier<Resolver> resolverSupplier)
+        {
+            resolver = ThreadLocal.withInitial(resolverSupplier);
+        }
+
+        @Override
+        public <R> R resolveExternal(long authorization, MessagePredicate filter, MessageFunction<R> mapper)
```

Comment:
Remember to apply coding conventions to all methods, with parameters on separate lines.